src/public/assets/lang/it_IT.yaml

```diff
@@ -39,37 +39,37 @@ open source on github: Open source su GitHub
-everyone knows dock/task bar. we got a brand new dock, with more features than ever before.: Tutti conoscono la dock/task bar. Noi abbiamo una nuova dock, con più funzionalità di prima.
-brand new launch menu: Nuovo launch menù
+everyone knows dock/task bar. we got a brand new dock, with more features than ever before.: Tutti conoscono la dock/task bar. Noi abbiamo una nuova dock, con più funzionalità che mai.
+brand new launch menu: Nuovo launch menu
 with everything in one place, do anything anywhere at anytime.: Con tutto nello stesso posto, fai di tutto ovunque e in ogni momento.
 easily apply layout: Applica facilmente il layout
-with a list of presets determined from you apps, easily get to work with the perfect window layout. not enough? change it in the settings.: Con un elenco di preset determinati dalle tue app, puoi facilmente lavorare con il tuo layout perfetto delle finestre. Non è abbastanza? Cambialo nelle impostazioni.
+with a list of presets determined from you apps, easily get to work with the perfect window layout. not enough? change it in the settings.: Con un elenco di preset determinati dalle tue app, lavora facilmente con il layout perfetto delle finestre. Non è abbastanza? Cambialo nelle impostazioni.
 overpowered dock: Dock potenziata
-your dock can do more than ever. it's your ultimate manager to get you organised.: La tua dock può fare più che mai. È il tuo massimo manager per organizzarti
+your dock can do more than ever. it's your ultimate manager to get you organised.: La tua dock può fare più che mai. È il tuo miglior manager per organizzarti.
 new way to manage files: Nuovo modo per gestire i file
 this file manager keeps you organised and productive. find your files the instant you need it.: Questo file manager ti mantiene organizzato e produttivo. Trova i tuoi file nell'istante in cui ne hai bisogno.
 drag & drop made easy: Drag & drop facile
-drag and drop is the easiest way to transfer anything on your computer. we make it intuitive and easy to use.: Il drag and drop è il modo più semplice per trasferire qualsiasi cosa sul tuo computer. Noi lo rendiamo intuitivo e facile da usare.
+drag and drop is the easiest way to transfer anything on your computer. we make it intuitive and easy to use.: Il drag and drop è il modo più semplice per trasferire qualsiasi cosa sul tuo computer. Noi lo abbiamo reso intuitivo e facile da usare.
 more than multitask: Più che multitasking
-want to do many tasks at a time? we know you and we got you. it's now not only multitasking, it's organised multitasking.: Vuoi fare più task alla volta? Lo sappiamo. Ora non è solo multitasking, è multitasking organizzato.
-tabs!: Tab!
+want to do many tasks at a time? we know you and we got you. it's now not only multitasking, it's organised multitasking.: Vuoi fare più task alla volta? Lo sappiamo e ti capiamo. Ora non è solo multitasking, è multitasking organizzato.
+tabs!: Schede!
```

Comment:
Sorry but I'm not sure why it says tabs! It should be "tabos!" and shows up on the website as "TabOS!".